src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -240,9 +244,7 @@ namespace winrt::Microsoft::Management::Deployment::implementation
                 }
             }
 
-            ::AppInstaller::Manifest::Manifest manifest;
-            manifest = GetManifestFromPackageVersionInfo(packageVersionInfo);
-            context.Add<::AppInstaller::CLI::Execution::Data::Manifest>(manifest);
+            AddPackageManifestToContext(packageVersionInfo, context);

```

Comment:
Adding the PackageVersion will become necessary to support dependencies in the future; maybe my last comment shouldn't have been a "nit" so much as a "forward facing request".